ClearScript/Util/MiscHelpers.cs

```diff
@@ -224,17 +224,36 @@ public static UIntPtr GetDigest(this string code)
             return (UIntPtr.Size == 4) ? (UIntPtr)code.GetDigestAsUInt32() : (UIntPtr)code.GetDigestAsUInt64();
         }
 
+        public static UIntPtr GetDigest(IntPtr code, int length)
```

Comment:
I would prefer to use `Span<byte>`, but it is not available in .NET 4.5.

Reply:
ClearScript 7.5 will use `Span`, at least internally. To do that on .NET Framework 4.5, it'll bring in [System.Memory 4.5.5](https://www.nuget.org/packages/System.Memory/4.5.5#supportedframeworks-body-tab).